src/Magento/FunctionalTestingFramework/Console/GenerateSuiteCommand.php

```diff
@@ -72,6 +72,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
         }
 
-        $output->writeLn("Suites Generated");
+        if ($this->cmdStatus) {
+            $output->writeLn("Suites Generated");
+            return 0;
+        } else {
+            $output->writeLn("Suite parsing error found. See mftf.log for details.");
+            $output->writeLn("Suites Generated");
```

Comment:
to indicate all tests in suite didn't generate, can the error message be tweaked:
`Suites Generated (partial)` or
`Suites Generated  (with failures)`

Reply:
Liked the 2nd version. Changed